lib/active_merchant/billing/gateways/bpoint.rb

```diff
@@ -154,7 +154,8 @@ def add_refund(xml, amount, transaction_number, options)
         transaction_number_xml(xml, transaction_number)
       end
 
-      def add_void(xml, amount, transaction_number, options)
+      def add_void(xml, transaction_number, options)
+        amount = options[:amount]
```

Comment:
In this case, is this a required option? I think so, just wanted to verify that.

Reply:
Yes, for the BPoint gateway, the `amount` is a required field for void transactions. This has been confirmed both through remote tests and an inquiry to BPoint's support staff. It's unusual for a void transaction to require this field, but BPoint apparently does. 🤷‍♀ 

Maybe a comment to confirm that would be helpful here?

Reply:
It doesn't hurt! But if it feels like overkill, I wouldn't sweat it. Thanks for that answer.